src/routes/formsgSiteLaunch.ts

```diff
@@ -229,11 +272,26 @@ export class FormsgSiteLaunchRouter {
         formResponses.map(this.launchSiteFromForm)
       )
       const successResults: DnsRecordsEmailProps[] = []
-      launchResults.forEach((launchResult) => {
+      for (const launchResult of launchResults) {
```

Comment:
> i get a big chunk of squiggles with the following warnings
weird i don't see this. but in any case for-of loops used with async-await is safe right? would you know more about the error?

Reply:
from [this](https://zellwk.com/blog/async-await-in-loops/) article, it's safe because we don't pass a callback as per `forEach`. The regnerator run time one is an eslint issue, there's a discussion for it but i'm not convinced it's impt as long as it's clear